index.html

```diff
@@ -10,43 +10,52 @@
 
         <header>
             <nav>
+                <ul class="top-nav">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
                 <img src="images/burgers.svg" alt="Menu-bar" class="hamburgers">
             </nav>
         </header>
+        <main class="main-section">
 
-        <section>
-            <div class="commence">
-                <h1 class="Introduction">
-
-                    Hey there. <br class="break">
-                    I'm Young. <br>
-                    <span class="green-h1">I'm a Software <br class="break">Developer</span>
-                </h1>
-                <div class="d-first-paragraph">
-                    <p class="first-paragraph"> I can help you build a product, feature or website Look through some of
-                        my work
-                        and experience! If you like what you see and have a project you need coded,
-                        don’t hesitate to contact me.</p>
-                </div>
+        <!-- <section> -->
+        <div class="commence">
+            <h1 class="Introduction">
+
+                Hey there. <br class="break">
+                I'm Young. <br>
+                <span class="green-h1">I'm a Software <br class="break">Developer</span>
+
+            </h1>
+
+            <div class="d-first-paragraph">
+
+                <p class="first-paragraph"> I can help you build a product, feature or website Look through some of
+                    my work
+                    and experience! If you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
 
             </div>
-            <ul class="social-media">
-                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
-                <li><a href="#"><img src="images/linkedin.svg" alt="linkeldin"></a></li>
-                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
-                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
-                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
-            </ul>
-            <div class="down-button">
-                <img src="images/down-button.svg" alt="down-button">
-            </div>
-    </main>
 
-    </section>
+        </div>
+        <ul class="social-media">
+            <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/linkedin.svg" alt="linkeldin"></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+            <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+        </ul>
+        <div class="down-button">
+            <img src="images/down-button.svg" alt="down-button">
```

Comment:
- [ ] I woul suggest yopu to stick to the original design as much posible to make your website look professional, Kindly fix the social media icons to be place at a good possition just like the original figma design ,you also give the icons a little margin top to under contents  
- [ ] your design

![Screenshot 2023-06-15 184924](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/4226123d-6062-406f-913f-a7ee1fc065ab)

- [ ] figma desig 
![fig design3](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/7b0c5eff-e821-45ee-9aa6-72c5823f3a48)